mews-operations/reservations.md

```diff
@@ -157,7 +157,11 @@ The third `reservation` definition shows the partial cancellation - cancelling t
                     "nationalityCode": "US",
                     "languageCode": "en-US",
                     "telephone": "1-369-81891",
-                    "loyaltyCode": "PG60972345"
+                    "loyaltyCode": "PG60972345",
+                    "loyaltyInfo": {
```

Comment:
If possible, can we use real (but anonymized) examples for LoyaltyInfo fields.

Reply:
yep, why not, do you know some? 😄